components/script/dom/windowproxy.rs

```diff
@@ -84,6 +88,7 @@ impl WindowProxy {
             reflector: Reflector::new(),
             browsing_context_id: browsing_context_id,
             top_level_browsing_context_id: top_level_browsing_context_id,
+            name: DomRefCell::new(DOMString::new()),
```

Comment:
To support the case of `<iframe src="whatever.html" name="hello">`, we need to do this:
```rust
let name = frame_element.map_or(DOMString::new(), |e| e.get_string_attribute(&local_name!("name")));
...
name: DomRefCell::new(name),
```